evm-symbolic.md

```diff
@@ -0,0 +1,26 @@
+EVM Execution
+=============
+
+This module contains custom extensions for smart contract verification.
+
+```k
+requires "edsl.k"
+
+module EVM-SYMBOLIC     [symbolic]
+    imports EVM
+    imports EDSL
+```
+
+Symbolic result of ecrecover.
+
+```k
+    //Symbolic wrapper over the argument of #ecrec, no implementation.
+    syntax Int ::= #symEcrec ( WordStack )   [function]
+
+    //Symbolic predicate representing whether output of #ecrec is empty. No implementation.
+    syntax Bool ::= #ecrecEmpty( WordStack ) [function]
+```
+
+```k
+endmodule
+```
```

Comment:
if we do not have much thing to add in this module, i think we can just move these two syntax declarations into EDSL.

Reply:
I added a few other things in the later commits. Lets keep it for now

Reply:
@ehildenb https://github.com/runtimeverification/verified-smart-contracts/pull/148/files